wsd/Admin.cpp

```diff
@@ -974,6 +974,16 @@ void Admin::addLostKitsTerminated(unsigned lostKitsTerminated)
     addCallback([this, lostKitsTerminated]{ _model.addLostKitsTerminated(lostKitsTerminated); });
 }
 
+void Admin::addKilledCount(unsigned killedCount)
```

Comment:
Hmm - we have three numbers - surely we can thread them through all this logic with a single method call, single callback etc. ? =) bearing in mind we're trying to add as little code as possible.

Reply:
Done